build.sh

```diff
@@ -0,0 +1,9 @@
+export DISPLAY=:0.0
+killall io.elementary.switchboard -9
+sudo rm -fr build/
+sudo meson build --prefix=/usr
+cd build/
+sudo ninja
+sudo ninja install
+cd ../
+io.elementary.switchboard
```

Comment:
I do not think this should be part of the repo.  If you need it in your source tree you should not add it to commits and set `.gitignore` to exclude it.

Personally I set up an alias in `~/.bashrc` to do something similar.

Reply:
> I do not think this should be part of the repo. If you need it in your source tree you should not add it to commits and set `.gitignore` to exclude it.
> 
> Personally I set up an alias in `~/.bashrc` to do something similar.

Will do

Reply:
> I do not think this should be part of the repo. If you need it in your source tree you should not add it to commits and set `.gitignore` to exclude it.
> 
> Personally I set up an alias in `~/.bashrc` to do something similar.

Thanks for reminding me about that, that would save some time.

Reply:
I think you need to revert this commit?